packages/core/src/blueprint-modern.scss

```diff
@@ -0,0 +1,692 @@
+
+    &:hover {
+      background: $orange4;
+    }
+
+    &:active {
+      background: $orange3;
+    }
+  }
+
+
+  &.#{$ns}-minimal {
+    @mixin pt-button-minimal-intent-modern($active-text-color) {
+      &:active {
+        color: $active-text-color;
+      }
+    }
+
+    @mixin pt-button-minimal-dark-intent-modern(
+      $hover-color,
+      $hover-text,
+      $active-color,
+      $active-text
+    ) {
+      &:hover {
+        background: $hover-color;
+        color: $hover-text;
+      }
+
+      &:active {
+        background: $active-color;
+        color: $active-text;
+      }
+    }
+
+    @each $intent, $colors in $button-intents {
+      &.#{$ns}-intent-#{$intent} {
+        @include pt-button-minimal-intent-modern(
+          map-get($button-active-minimal-intent-text-modern, $intent),
+        );
+      }
+    }
+
+    .#{$ns}-dark & {
+      &:not([class*="#{$ns}-intent-"]) {
+        color: $white;
+      }
+
+      @each $intent, $color in $dark-minimal-button-intents-modern {
+        &.#{$ns}-intent-#{$intent} {
+          @include pt-button-minimal-dark-intent-modern($color...);
+        }
+      }
+    }
+  }
+}
+
+// Contrast for callouts
+.#{$ns}-callout {
+  background-color: rgba($gray3, 0.15);
+
+  .#{$ns}-dark & {
+    background-color: rgba($gray3, 0.2);
+
+    &.#{$ns}-callout-icon:not([class*="#{$ns}-intent-"])>.#{$ns}-icon:first-child {
+      color: $pt-dark-icon-color;
+    }
+  }
+
+  @each $intent, $color in $pt-intent-colors {
+    &.#{$ns}-intent-#{$intent} {
+      background-color: rgba($color, 0.1);
+
+      .#{$ns}-dark & {
+        background-color: rgba($color, 0.2);
+      }
+    }
+  }
+}
+
+// Contrast for checkboxes and radio buttons
+.#{$ns}-control {
+  input:focus ~ .#{$ns}-control-indicator {
+    outline: $blue3 auto 2px;
+  }
+
+  &.#{$ns}-checkbox, &.#{$ns}-radio {
+    .#{$ns}-control-indicator {
+      box-shadow:
+        inset 0 0 0 $button-border-width rgba($black, 0.5),
+        //TODO: verify that this is right
+        inset 0 (-$button-border-width) 0 rgba($black, 0.1);
+    }
+  }
+
+  .#{$ns}-dark & {
+    input:focus ~ .#{$ns}-control-indicator {
+      outline: $blue5 auto 2px;
+    }
+
+    &.#{$ns}-checkbox, &.#{$ns}-radio {
+      input:not(:disabled):active ~ .#{$ns}-control-indicator {
+        background:
+          linear-gradient(0deg, rgba($white, 0.1), rgba($white, 0.1)),
+          linear-gradient(0deg, $gray2, $gray2);
+      }
+    }
+  }
+}
+
+// TODO
+// Contrast for date pickers
+
+// Contrast for HTML tables
+table.#{$ns}-html-table {
+  &.#{$ns}-html-table-striped {
+    tbody tr:nth-child(odd) td {
+      background: rgba($gray3, 0.15);
+    }
+  }
+
+  &.#{$ns}-interactive {
+    tbody tr {
+      &:hover td {
+        background-color: rgba($gray3, 0.3);
+        cursor: pointer;
+      }
+
+      &:active td {
+        background-color: rgba($gray3, 0.4);
+      }
+    }
+  }
+}
+
+// Contrast for icons
+.#{$ns}-icon:not([class*="#{$ns}-intent-"]) {
```

Comment:
By default the non-intent colors use `$pt-text-color` instead of `$pt-icon-color` (i.e. `$pt-text-color-muted`) and likewise for dark mode, because they were being imported from `body`:
![Screen Shot 2021-09-21 at 15 33 25](https://user-images.githubusercontent.com/5934779/134235882-89ff8cb0-dde1-490b-8c60-bde925231aa4.png)

I've overwritten the color here so that we don't just default to the default text colors/to match the colors that are in the proposed designs, but I'm not sure if this will break anything down the line. I.e., if we're breaking icon color inheritance somewhere else

Reply:
Icons _are_ supposed to inherit text color in this way, so yeah, I think this could break something further up the application stack.

What was the goal of setting this particular line with `color`? Where do we see it in effect?

Reply:
This was originally to fix the info icon for callouts, dialogs, and toasts. I was able to fix these in each of those components so this is no longer necessary

<img width="504" alt="Screen Shot 2021-09-23 at 14 17 07" src="https://user-images.githubusercontent.com/5934779/134561956-fe4a115d-bc59-470d-bac0-68f8cf96c499.png">

<img width="504" alt="Screen Shot 2021-09-23 at 14 16 57" src="https://user-images.githubusercontent.com/5934779/134561947-9838ec32-80cf-44ee-8ff5-ca491c8c27ac.png">
<img width="1328" alt="Screen Shot 2021-09-23 at 14 16 35" src="https://user-images.githubusercontent.com/5934779/134561941-5c182ea3-c633-42ff-bfc6-df51d4a08a4c.png">
<img width="1336" alt="Screen Shot 2021-09-23 at 14 18 15" src="https://user-images.githubusercontent.com/5934779/134562055-79250a47-9f20-432a-a603-e5998d8a2bb1.png">